EnlightenMAUI.csproj

```diff
@@ -77,6 +77,7 @@
 	</ItemGroup>
 
 	<ItemGroup>
+	  <None Remove="libraries\SiG-785-OEM\Acetaminophen.json" />
```

Comment:
I guess I'm curious why this goes in an OEM folder but okay

Reply:
to generically separate from the previous library files